doc/src/site/sphinx/ml/sw_deep_learning.rst

```diff
@@ -0,0 +1,114 @@
+Train Deep Learning Model in Sparkling Water
+--------------------------------------------
+
+Sparkling Water provides API for H2O Deep Learning in Scala and Python. The following sections describe how to train
+the Deep Learning model in Sparkling Water in both languages. See also :ref:`parameters_H2ODeepLearning`.
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        First, let's start Sparkling Shell as
+
+        .. code:: shell
+
+            ./bin/sparkling-shell
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: scala
+
+            import ai.h2o.sparkling._
+            import java.net.URI
+            val hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: scala
+
+	        import org.apache.spark.SparkFiles
+            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+	        val rawSparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
+            val sparkDF = rawSparkDF.withColumn("CAPSULE", $"CAPSULE" cast "string")
+            val Array(trainingDF, testingDF) = sparkDF.randomSplit(Array(0.8, 0.2))
+
+        Train the model. You can configure all the available DeepLearning arguments using provided setters, such as the label column
+        or the layout of hidden layers.
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.algos.H2ODeepLearning
+            val estimator = new H2ODeepLearning().setLabelCol("CAPSULE").setHidden(Array(3, 2))
+            val model = estimator.fit(trainingDF)
+
+        By default, the ``H2ODeepLearning`` algorithm distinguishes between a classification and regression problem based on the type of
+        the label column of the training dataset. If the label column is a string column, a classification model will be trained.
+        If the label column is a numeric column, a regression model will be trained. If you don't want be worried about
+        column data types, you can explicitly identify the problem by using ``ai.h2o.sparkling.ml.algos.classification.H2ODeepLearningClassifier``
```

Comment:
Change

you can explicitly identify the problem by 

to you can explicitly specify the model by